4617_Sprint/3-Sheik/Demo.java

```diff
@@ -0,0 +1,64 @@
+import java.time.LocalDate;
+import java.time.Period;
+import java.util.Scanner;
+
+public class Demo {
+
```

Comment:
Pretty much everything could be private instead of public, other then that looking good